docs/README.md

```
@@ -1761,6 +1762,41 @@ osdctl cluster hypershift-info [flags]
+
+Migrate ROSA Classic cluster nodes to enforce IMDSv2.
+
+This automates the SOP for migrating machines to IMDSv2 by:
+- Patching Hive MachinePools to require IMDSv2
+- Replacing infra nodes (one at a time)
+- Updating ControlPlaneMachineSet for automatic master node rollout
+- Validating all nodes/machines are using IMDSv2
+
+Pre-flight checks verify cluster health before making changes.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)